CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# Will be requested for
+# review when someone opens a pull request.
+* @yugabyte/ybm-cli-owners
```

Comment:
@shinji62 do you plan to create this file later?

Reply:
This file says that for changes to all files and folders under (`*`) require approval from `yugabyte/ybm-cli-owners`.